packages/svelte/src/reactivity/index-server.js

```diff
@@ -1,5 +1,8 @@
-export const Date = globalThis.Date;
-export const Set = globalThis.Set;
-export const Map = globalThis.Map;
-export const URL = globalThis.URL;
-export const URLSearchParams = globalThis.URLSearchParams;
+export const SvelteDate = globalThis.Date;
+export const SvelteSet = globalThis.Set;
+export const SvelteMap = globalThis.Map;
+export const SvelteURL = globalThis.URL;
+export const SvelteURLSearchParams = globalThis.URLSearchParams;
+
+// Deprecated
+export { Date, Set, Map, URL, URLSearchParams } from './index-client.js';
```

Comment:
On second thought, do we know that this is going to treeshake well? Do we just want to copy over the functions that throw the 'x has been removed' errors?

Reply:
Done